packages/ui/src/components/viewport/EditorPreview.tsx

```diff
@@ -246,6 +252,8 @@ export function EditorPreview() {
             <GridIcon />
           </ButtonCheckbox>
           <ColorPicker />
+          {/* Key is set to the state so that the component renders when the state changes */}
```

Comment:
Please elaborate

Reply:
Yeah I got some cleanup to do. That was a remnant comment after I changed the implementation. It is no longer required. I will remove.